README.md

```diff
@@ -1,69 +1,108 @@
-# ![nf-core/circrna](docs/images/nf-core-circrna_logo.png)
+# ![nf-core/circrna](docs/images/nf-core-circrna_logo_light.png#gh-light-mode-only) ![nf-core/circrna](docs/images/nf-core-circrna_logo_dark.png#gh-dark-mode-only)
 
-**workflow for the quantification, differential expression analysis and miRNA target prediction analysis of circRNAs in RNA-Seq data**.
+[![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?labelColor=000000&logo=Amazon%20AWS)](https://nf-co.re/circrna/results)[![Cite with Zenodo](http://img.shields.io/badge/DOI-10.5281/zenodo.XXXXXXX-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.XXXXXXX)
```

Comment:
@jfy133 @apeltzer can we figure this out before the relase? Otherwise, another Zenodo ID is coming ;) Barry, the docs for this were recently updated. Shout if you need help to add the ID after the release. Someone from core (Alex) needs to generate this beforehand for you.